configuration/pih/htmlforms/mentalHealth.xml

```diff
@@ -917,6 +917,31 @@
 
             </div>
         </div>
+
+           <div class="section-container" id="radio_button_container">
+
+            <div class="col">
+                <!-- <p>
+                    <label>
+                        comment
+                    </label>
+                    <obs conceptId="CIEL:163044" type="textarea"/>
+                </p> -->
+                <p class="comment-label"> <uimessage code="pihcore.exam.comment"/> </p> 
+
+                    <p class="comment-field"> 
+
+                        <p> 
+                            <obs conceptId="CIEL:163044" rows="5" cols="25"/> 
```

Comment:
@Dutervil You are already using this concept (CIEL:163044) on line 907 of this file.  You cannot use the same concept twice unless it's grouped in an obsgroup/convenience set (ConvSet).

Reply:
@lnball What is the right concept to use, please?

Reply:
@lnball I was not sure about the conpcet , i did it while you were in holiday, Please let me know the right ticket to use pleae